<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1353</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>1571</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/new/prefix1/Nouveau-projet-_2_.png);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>ajouter </string>
    </attribute>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>veillez entrer les donnes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prenom">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>date de naissance</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_grade">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>grade</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_grade">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>460</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_id">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>identifant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_id">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_lieu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>lieu de travail</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_lieu_t">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>310</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_tele">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>telephone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_prenom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_nom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="ajouteremail">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>390</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="chercher">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>570</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>chercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>570</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 255, 127);
background-color: rgb(0, 255, 127);</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifier">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>570</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>570</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 31, 2);
selection-color: rgb(255, 38, 0);
color: rgb(255, 26, 6);</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="trierecher">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>500</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>salaire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>id</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="tri">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>500</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>trier selon:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_tele">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_email">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>380</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_salaire">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria Math</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>salaire</string>
     </property>
    </widget>
    <widget class="QTableView" name="afficher_t">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>861</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="l_date">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>8</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>2000</year>
       <month>8</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2000</year>
       <month>8</month>
       <day>14</day>
      </datetime>
     </property>
     <property name="maximumDate">
      <date>
       <year>2000</year>
       <month>8</month>
       <day>14</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2000</year>
       <month>8</month>
       <day>14</day>
      </date>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
     <property name="date">
      <date>
       <year>2000</year>
       <month>8</month>
       <day>14</day>
      </date>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 77, 156);
border-color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>          GESTION DES PERSONNELS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trier">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>500</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>trier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>mot_de_passe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="motpasse_l">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>500</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>570</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>pdf downlowd</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="l_gender_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>530</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="l_gender">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>69</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>gender</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>_preview impression</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1041</width>
       <height>621</height>
      </rect>
     </property>
     <property name="title">
      <string>PDF</string>
     </property>
     <widget class="QPushButton" name="imprimer">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>520</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>520</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>pdf_preview</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>761</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>statistique</string>
    </attribute>
    <widget class="QPushButton" name="stat">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>330</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
